node/test/Test/Pos/CryptoSpec.hs

```diff
@@ -111,13 +102,16 @@ spec = describe "Crypto" $ do
                 binaryTest @Crypto.EncShare
                 binaryTest @Crypto.SecretProof
                 binaryTest @Crypto.SecretSharingExtra
+                binaryTest @Crypto.HDAddressPayload
+                binaryTest @(Crypto.AbstractHash SHA256 U)
```

Comment:
Are we using SHA256 anywhere?

Reply:
Valid concern. It was in `CborSpec` so i just moved it here. Probably should be removed.